### PR TITLE
108 update so solo movies tab only shows logged in user

### PR DESCRIPTION
**Pull request by beta9889**
only logged in user. 
keept api functionality if username is not sent in request

### CODE REVIEW

File: api/models/solo_movies.php
Comment (microl44) [inline api/models/solo_movies.php:1]: Please git restore line removal changes if it's the only thing changed.

File: includers/basic.js
Comment (microl44) [inline includers/basic.js:1]: Good change

File: Pages/index.js
Comment (microl44) [inline Pages/index.js:1]: GetCookie function already exists on line 113, why is this not used?

Comment (microl44) [inline Pages/index.js:1]: FetchSolo should probably not be called in function which returns active table name.

Comment (microl44) [inline Pages/index.js:1]: Consol.log(response) should be kept in. 

Useful for debugging.

File: Pages/index.js [hunk 1]
```diff
@@ -1052,24 +1051,47 @@ function CreateLoadingRow()
   //table.append(loadingDiv);
 }
 
+function getCookie(cname) {
```
[the remark this answers is not in the document]
Comment (beta9889) [reply Pages/index.js:1054]: removed

File: Pages/index.js [hunk 2]
```diff
@@ -143,6 +143,7 @@ function GetActiveTableName()
         retval = 'Group';
         break;
       case 'soloMovie':
+        FetchSolo();
```
[the remark this answers is not in the document]
Comment (beta9889) [reply Pages/index.js]: moved

File: api/solo_movies.php [hunk 3]
```diff
@@ -6,25 +6,60 @@
...
+	$results['data'] = array();
+	$sql = 'SELECT solo_movie_participants.id,`participant id`,`participant`,`user rating`,`imdb rating`,`movie`,`description`,`cover path` 
+		FROM solo_movie_participants
+		HAVING Participant=:username' ;
+	$stmt = $conn->prepare($sql);
+	$stmt->bindvalue(':username', $username,PDO::PARAM_STR);
+	return $stmt;
+}
+
+function setupStmtForGettingAll()
```
[the remark this answers is not in the document]
Comment (beta9889) [reply api/solo_movies.php:22]: ok boss

**Pull request merged**